src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -76,12 +76,14 @@ class Sales extends React.Component<Props, State> {
       },
     ]
 
+    // The `renderItem` is alway
```

Comment:
😄

Reply:
^

Reply:
@orta I guess you probably wanted to leave a note about why `renderItem` is set to be `() => undefined` ?